src/Agent.Worker/Handlers/LegacyPowerShellHandler.cs

```diff
@@ -177,6 +177,9 @@ public async Task RunAsync()
             ArgUtil.NotNull(Inputs, nameof(Inputs));
             ArgUtil.Directory(TaskDirectory, nameof(TaskDirectory));
 
+            // Warn about legacy handler.
+            ExecutionContext.Warning($"Task '{this.Task.Name}' ({this.Task.Version}) is using deprecated task execution handler, stop use this task and contact task author to upgrade their task.");
```

Comment:
Task '{this.Task.Name}' ({this.Task.Version}) is using deprecated task execution handler.  The task should use the supported task-lib: https://aka.ms/tasklib

Reply:
Date per Matt Cooper